src/platform/forms-system/src/js/components/ConfirmationView/ChapterSectionCollection.jsx

```diff
@@ -0,0 +1,329 @@
+import React from 'react';
```

Comment:
This file is migrated from https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/simple-forms/shared/components/confirmationPageViewHelpers.jsx. Minimal changes here, you can compare the signature on line 264.